### DISCUSSION TITLE
why data alignment is important for speed?

### DISCUSSION THREAD

**Discussion by liusy58** [liusy58/WhyThis#41] [General]
RT

**Comment by liusy58** [comment]
一般都会扯到需要两次read然后重组所以会比较慢，但是通过很多的实验发现其实并不会慢，但是很多书还是保留了data alignment对于speed的重要性，具体的话还需要进一步的研究。

**Reply by uniform641** [reply]
我觉得这里还可以考虑多核情况下False Sharing带来的性能开销问题。

**Reply by liusy58** [reply]
我稍后写一篇文章
